spec/system/post_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  describe 'Index' do
+    before(:all) do
+      @user = User.first
+      @posts = @user.posts.all
+    end
+
+    it 'I can see the user profile picture' do
+      visit user_posts_path(@user)
+      expect(page).to have_css("img[src*=\"#{@user.photo}\"]")
+    end
+
+    it 'I can see the username' do
+      visit user_posts_path(@user)
+      expect(page).to have_content(@user.name)
+    end
+
+    it 'I can see the number of posts the user has written' do
+      visit user_posts_path(@user)
+      expect(page).to have_content(@user.posts.count)
+    end
+
+    it 'I can see a post title' do
+      visit user_posts_path(@user)
+      @posts.each do |post|
+        expect(page).to have_content(post.title)
+      end
+    end
```

Comment:
- [ ] This particular test is failing as you can see below 👇 Please try to fix it
![test fail](https://user-images.githubusercontent.com/46839548/217663797-74f50118-8bc6-481f-a104-b13cdc62d536.PNG)

Reply:
![corrected](https://user-images.githubusercontent.com/105460631/217672787-4d355f58-fa87-46be-a852-4035018bb310.PNG)